numpy/f2py/_backends/src/meson.build.src

```diff
@@ -0,0 +1,40 @@
+project('${modulename}',
+        ['c', 'fortran'],
+        version : '0.1',
+        meson_version: '>= 1.1.0',
+        default_options : ['warning_level=2'])
```

Comment:
Is this necessary? A higher warning level is typically not desired I think.

Reply:
Also, did you think about what the default build type should be? Perhaps it should be `release`?

Reply:
> Is this necessary? A higher warning level is typically not desired I think.

Ah good point, I see the [default is](https://mesonbuild.com/Builtin-options.html#core-options) now $1$, when I wrote the `f2py` with `meson` [documentation](https://numpy.org/doc/stable/f2py/buildtools/meson.html) it defaulted to $3$ so this was a lowering done to stop passing `-fimplicit-none`. I'll explicitly set it to $1$ then, just in case the defaults change later.

Reply:
> Also, did you think about what the default build type should be? Perhaps it should be `release`?

It should respect `f2py`'s `--debug` flag, updating.